packages/vsce/src/trees/CICSTree.ts

```diff
@@ -75,63 +78,118 @@ export class CICSTree implements TreeDataProvider<CICSSessionTree> {
   }
 
   /**
-   *
-   * Provides user with prompts and allows them to add a profile after clicking the '+' button
-   */
+     * Provides user with user actions and allows them to manage the selected profile
+     * @param treeview CICSTree View
+     * *@param node current selected node
+     */
+  async manageProfile(treeview: TreeView<any>, node: any) {
+    const allSelectedNodes = findSelectedNodes(treeview, CICSSessionTree, node);
+    if (!allSelectedNodes || !allSelectedNodes.length) {
+      window.showErrorMessage("No profile selected to delete");
+      return;
+    }
+    try {
+      const configInstance = await ProfileManagement.getConfigInstance();
+      if (configInstance.getTeamConfig().exists) {
+        const currentProfile = await ProfileManagement.getProfilesCache().getProfileFromConfig(
+          allSelectedNodes[allSelectedNodes.length - 1].label,
+        );
+
+        const deleteProfile: vscode.QuickPickItem = {
+          label: `$(trash) ${vscode.l10n.t("Delete Profile")}`,
+          description: vscode.l10n.t("Delete the selected Profile"),
```

Comment:
Can we change these labels / descriptions to be single or plural depending on if multiple profiles are selected?? Something like...

```suggestion
          label: `$(trash) ${vscode.l10n.t(`Delete Profile${allSelectedNodes.length > 1 ? "s" : ""}`)}`,
          description: vscode.l10n.t(`Delete the selected Profile${allSelectedNodes.length > 1 ? "s" : ""}`)}`),
```

Reply:
Adopted this suggestion..!!
Also, modified Hide and Update messages as per this suggestion.